recipes/pydantic-config/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pydantic-config" %}
+{% set version = "0.2.3" %}
+{% set repo_url = "https://github.com/jordantshaw/pydantic-config" %}
```

Comment:
I'd rather have these explicit in the fields below than having it here as a jinja variable. The name and version makes sense b/c our bots use them but URLs are for humans and I'd like to click on them when reading the metadata in the recipe instead of searching for it in the file.

Reply:
@ocefpaf  makes sense. Pushed those changes.